src/Resources/translations/SonataDashboardBundle.en.xliff

```diff
@@ -290,6 +290,30 @@
                 <source>sonata_dashboard</source>
                 <target>Dashboard</target>
             </trans-unit>
+            <trans-unit id="73" resname="breadcrumb.link_dashboard_block_batch">
+                <source>breadcrumb.link_dashboard_block_batch</source>
+                <target>Batch</target>
+            </trans-unit>
+            <trans-unit id="74" resname="breadcrumb.link_dashboard_block_create">
+                <source>breadcrumb.link_dashboard_block_create</source>
+                <target>Create</target>
+            </trans-unit>
+            <trans-unit id="75" resname="breadcrumb.link_dashboard_block_delete">
+                <source>breadcrumb.link_dashboard_block_delete</source>
+                <target>Delete</target>
+            </trans-unit>
+            <trans-unit id="76" resname="breadcrumb.link_dashboard_block_edit">
+                <source>breadcrumb.link_dashboard_block_edit</source>
+                <target>Edit</target>
+            </trans-unit>
+            <trans-unit id="77" resname="breadcrumb.link_dashboard_block_list">
+                <source>breadcrumb.link_dashboard_block_list</source>
+                <target>Blocks list</target>
```

Comment:
Shouldn't this be a verb like the other actions? Mean, "List blocks" or just "List".

Reply:
No, it's just the list of all blocks :trollface: 

It's the same like this: https://github.com/sonata-project/SonataPageBundle/blob/3.x/src/Resources/translations/SonataPageBundle.en.xliff#L25-L28

Reply:
I guess I understand, maybe this is a discussion for a more specific PR. But just to share my point of view, let me explain my reasoning.
There are other translations previous this one, expressed in the form of a verb which trigger an action.
IE:
* **Edit** (verb) link will redirect the user to the **edit form** (subject)
* **Delete** (verb) link will redirect the user to the **delete confirmation form** (subject)

So, to my understanding, the "list" action should be expressed in the same way:
* **List** (verb) link will redirect the user to the **blocks list** (subject)

Reply:
Changed this to `Blocks`